mod-source-record-manager-server/src/main/java/org/folio/verticle/consumers/RawMarcChunksKafkaHandler.java

```diff
@@ -50,8 +50,7 @@ public RawMarcChunksKafkaHandler(@Autowired @Qualifier("eventDrivenChunkProcessi
 
     try {
       RawRecordsDto rawRecordsDto = new JsonObject(ZIPArchiver.unzip(event.getEventPayload())).mapTo(RawRecordsDto.class);
-      LOGGER.debug("RawRecordsDto has been received, starting processing correlationId: {} chunkNumber: {} - {}", correlationId,
-        chunkNumber, rawRecordsDto.getRecordsMetadata());
+      LOGGER.debug("RawRecordsDto has been received, starting processing correlationId: {} chunkNumber: {} - {}", correlationId, chunkNumber, rawRecordsDto.getRecordsMetadata());
```

Comment:
@SerhiiNosko could you please add jobExecutionId to log msg, it can be useful to distinguish import sessions
jobExecutionId can be obtained from okapiConnectionParams above in this way: 
String jobExecutionId = okapiConnectionParams.getHeaders().get("jobExecutionId");

```suggestion
      LOGGER.debug("RawRecordsDto has been received, starting processing jobExecutionId: {} correlationId: {} chunkNumber: {} - {}", jobExecutionId, correlationId,
```

Reply:
added, thanks

Reply:
Thank you @SerhiiNosko!